mlflow/utils/validation.py

```diff
@@ -17,6 +17,14 @@
     " spaces ( ), and slashes (/)."
 )
 
+MAX_PARAMS_TAGS_PER_BATCH = 100
```

Comment:
👍 

Does it make sense to have backend stores like `SqlAlchemyStore` uses these constants for sql validation in definitions schema?